README.md

```diff
@@ -163,6 +163,15 @@ $ dm pkg import-all <path> <data_source>
 $ dm pkg import-all app/data/DemoApplicationDataSource DemoApplicationDataSource
 ```
 
+
+Exporting document(s) in a package as a zip file
+```sh
+# export document(s) in a package in the directory given by <path>
+$ dm pkg export <path> <data_source>
+# Export all documents in 'app/data/DemoApplicationDataSource'
+$ dm pkg export app/data/DemoApplicationDataSource DemoApplicationDataSource
```

Comment:
Should be changed to: 

```dm pkg export DemoApplicationDataSource/models/CarPackage```

```dm pkg export <data_source>/<path>``` 
```dm pkg export <data_source>/<path> <where to store>``` 
```dm pkg export --expand --override <data_source>/<path> <where to store>``` 

Also provide the option to:
- Be able to specify where to store the result, default should be where you run the command from
- Only download Zip file by default, but can add flag also to expand

NOTE: Option to override the folder